content/courses/program-security/arbitrary-cpi.md

```diff
@@ -13,20 +13,19 @@ description: "How to safely invoke Solana programs from other Solana programs."
 ## Summary
 
 - To generate a CPI, the target program must be passed into the invoking
-  instruction as an account. This means that any target program could be passed
-  into the instruction. Your program should check for incorrect or unexpected
-  programs.
-- Perform program checks in native programs by simply comparing the public key
-  of the passed-in program to the progam you expected.
-- If a program is written in Anchor, then it may have a publicly available CPI
+  instruction as an account. Any target program could be passed in, so your
+  program should check for incorrect or unexpected programs.
+- Perform program checks in native programs by comparing the public key of the
+  passed-in program to the program you expected.
+- If a program is written in Anchor, it may have a publicly available CPI
```

Comment:
What is a 'publicly available CPI'? Does it mean 'publicly available IDL' maybe? Or am I missing something?

Reply:
It is publicly available cross program invocations no?